gum/gumeventsink.h

```diff
@@ -16,6 +16,10 @@ G_BEGIN_DECLS
 #define GUM_TYPE_EVENT_SINK (gum_event_sink_get_type ())
 G_DECLARE_INTERFACE (GumEventSink, gum_event_sink, GUM, EVENT_SINK, GObject)
 
+#define GUM_TYPE_DEFAULT_EVENT_SINK (gum_default_event_sink_get_type ())
+G_DECLARE_FINAL_TYPE (GumDefaultEventSink, gum_default_event_sink, GUM,
+                      DEFAULT_EVENT_SINK, GObject)
```

Comment:
```suggestion
    DEFAULT_EVENT_SINK, GObject)
```

Hanging indents should be 4 spaces. (Just realised we're already a bit inconsistent when using these macros, ugh.)